src/command_manager.cpp

```diff
@@ -103,17 +114,38 @@ void CommandManager::init_failsafe()
 void CommandManager::interpret_rc(void)
 {
   // get initial, unscaled RC values
-  rc_command_.x.value = RF_.rc_.stick(RC::STICK_X);
-  rc_command_.y.value = RF_.rc_.stick(RC::STICK_Y);
-  rc_command_.z.value = RF_.rc_.stick(RC::STICK_Z);
-  rc_command_.F.value = RF_.rc_.stick(RC::STICK_F);
+  // TODO: Adjust this to choose the channel that the RC thottle corresponds to
```

Comment:
Does this TODO need to be resolved or does the feature work and this will be dealt with later?

Reply:
The feature works, but I forgot to remove the TODO